code/processes/dqc.q

```diff
@@ -219,10 +222,14 @@ reruncheck:{[chkid]
   .timer.removefunc'[exec funcparam from .timer.timer where `.dqe.writedown in' funcparam];                     /- clear writedown timer
   .timer.removefunc'[exec funcparam from .timer.timer where `.dqe.writedownconfig in' funcparam];               /- clear writedownconfig timer
   delete configtable from `.dqe;
-  .dqe.init[];
   .dqe.currentpartition:pt+1;
+  if[(`timestamp$.dqe.currentpartition)>=.eodtime.nextroll;.eodtime.nextroll:.eodtime.getroll[`timestamp$.dqe.currentpartition];.lg.o[`dqc;"Moving .eodtime.nextroll to match current partition"]]
```

Comment:
For the sake of readability split this up + add a semi colon to the end as it is missing + add a comment to explain why this statement exists:
```suggestion
  if[(`timestamp$.dqe.currentpartition)>=.eodtime.nextroll;
    .eodtime.nextroll:.eodtime.getroll[`timestamp$.dqe.currentpartition];
    .lg.o[`dqc;"Moving .eodtime.nextroll to match current partition"]
  ];
```